frontend/test/__support__/cypress.js

```diff
@@ -267,6 +267,17 @@ export function expectedRouteCalls({ route_alias, calls } = {}) {
   });
 }
 
+export function remapDisplayValueToFK(display_value, fk) {
+  // Both display_value and fk are expected to be field IDs
+  // You can get them from frontend/test/__support__/cypress_sample_dataset.json
+  cy.request("POST", `/api/field/${display_value}/dimension`, {
+    field_id: display_value,
+    name: "Product ID",
```

Comment:
Woops

Reply:
Ooopsy, indeed.
To be able to access the names, I'll need to make this function A LOT more complicated. It needs a wrapper that basically access the whole `SAMPLE_DATASET` (to be able to reference tables), then some filtering...

Not sure if it's worth the hassle?

@paulrosenzweig do you maybe have some ideas? This is the scenario I asked you about on Slack - how can children "know" about their parent (object).

Reply:
Or (to keep things simple) should I simply use object destructuring here and provide `name` as an argument? Like this:

```javascript
export function remapDisplayValueToFK({display_value, name, fk} = {}) {
  // Both display_value and fk are expected to be field IDs
  // You can get them from frontend/test/__support__/cypress_sample_dataset.json
  cy.request("POST", `/api/field/${display_value}/dimension`, {
    field_id: display_value,
    name
    human_readable_field_id: fk,
    type: "external",
  });
}
```

Use it like this:
`remapDisplayValueToFK({ display_value: ORDERS.PRODUCT_ID, name: "Product ID", fk: PRODUCTS.TITLE})`

Reply:
For now I would keep it simple, and get it working just by doing something like this:
```
export function remapDisplayValueToFK(display_value, fk, remapName) {
    name: remapName,
```
It would be great to have it dynamically, but the root cause of setting the name during the remapping is actually issue 5558

Reply:
Hmmm - Github didn't update with your comment - go with that if it works ;-)

Reply:
I'd REALLY like to avoid people guessing the order of the arguments. That's why I lean towards destructured version.